package.json

```diff
@@ -57,6 +57,7 @@
     "jest": "^26.6.1",
     "lerna-changelog": "^0.5.0",
     "lint-staged": "^9.2.0",
+    "lodash": "^4.17.20",
```

Comment:
Is this unrelated?

Reply:
Cherry picked from #12011 because I built babel-parser in pnp mode.